resources/views/inputs/includes/markdown/navbar.blade.php

```diff
@@ -0,0 +1,113 @@
+@props(['mobile' => false, 'toolbar'])
+
+<div
+    @class([
+        'toastui-editor-toolbar ark-markdown-editor-toolbar dark:bg-theme-secondary-800',
+        'flex relative flex-col whitespace-nowrap' => !$mobile,
+        'ark-markdown-editor-toolbar-mobile' => $mobile,
+    ])
+    @if($mobile)
+        x-cloak
+        x-show="showMobileMenu"
+    @endif
+>
+    <div
+        @class([
+            'flex items-center toastui-editor-toolbar-group px-2',
+            'overflow-hidden z-10 relative' => !$mobile,
+            'flex-wrap z-20 bg-white dark:bg-theme-secondary-800 rounded-xl absolute shadow-2xl border-none -mx-4 px-5 py-6 right-0' => $mobile,
+        ])
+
+    >
+        @include('ark::inputs.includes.markdown.button', ['name' => 'undo', 'iconName' => 'undo', 'onClick' => 'undo'])
+
+        @include('ark::inputs.includes.markdown.button', ['name' => 'redo', 'iconName' => 'redo', 'onClick' => 'redo'])
+
+        @include('ark::inputs.includes.markdown.button-separator')
+
+        @include('ark::inputs.includes.markdown.button', ['name' => 'bold', 'iconName' => 'text-bold', 'onClick' => 'strong', 'nodeName' => 'strong'])
+
+        @include('ark::inputs.includes.markdown.button', ['name' => 'italic', 'iconName' => 'text-italic', 'onClick' => 'emph', 'nodeName' => 'emph'])
+
+        @unless ($toolbar === 'basic')
+            @include('ark::inputs.includes.markdown.button', ['name' => 'strike', 'iconName' => 'text-strike-through', 'onClick' => 'strike', 'nodeName' => 'strike'])
+
+            @include('ark::inputs.includes.markdown.button', ['name' => 'underline', 'iconName' => 'text-underline', 'onClick' => 'underline', 'nodeName' => 'underline'])
+
+            @include('ark::inputs.includes.markdown.button', ['name' => 'blockquote', 'iconName' => 'open-quote', 'onClick' => 'blockQuote', 'nodeName' => 'blockQuote'])
+
+            @include('ark::inputs.includes.markdown.button-separator')
+
+            @for($i=1; $i<=4; $i++)
+                @include('ark::inputs.includes.markdown.button', [
+                    'tooltip' =>  trans('ui::markdown.navbar.tooltips.heading', ['level' => $i]),
+                    'iconName' => 'H' . $i,
+                    'onClick' => 'heading(' . $i . ')',
+                    'nodeName' => 'heading' . $i,
+                ])
+            @endfor
+        @endunless
+
+        @include('ark::inputs.includes.markdown.button-separator')
+
+        @include('ark::inputs.includes.markdown.button', ['name' => 'ordered_list', 'iconName' => 'list-bullets', 'onClick' => 'bulletList', 'nodeName' => 'bulletList'])
+
+        @include('ark::inputs.includes.markdown.button', ['name' => 'unordered_list', 'iconName' => 'list-numbers', 'onClick' => 'orderedList', 'nodeName' => 'orderedList'])
```

Comment:
The ordered list should be numbered, the unordered list should be bullets. It's currently the wrong way around

![image](https://user-images.githubusercontent.com/35610748/148027819-5923f3fb-1d17-4920-bba1-c63f536bf7eb.png)